src/parameters/Parameters.jl

```diff
@@ -51,6 +75,15 @@ end
 
 Base.eltype(::ClimaAtmosParameters{FT}) where {FT} = FT
 
+thermodynamics_params(ps::ATCP) = ps.thermodynamics_params
+surface_fluxes_params(ps::ATCP) = ps.surface_fluxes_params
```

Comment:
```suggestion
```

I don't think these are needed, they should auto-generated by this snippet below
```
for var in fieldnames(ClimaAtmosParameters)
    @eval $var(ps::ACAP) = ps.$var
end
```